README.md

```diff
@@ -31,11 +31,7 @@ That said, keep in mind the following:
 
 ## Setup
 
-Get the tool via
-
-```sh
-$ go get -u github.com/golang/dep/cmd/dep
-```
+Grab the latest binary from the [releases](https://github.com/golang/dep/releases) page.
```

Comment:
Will `go get` be the normal users installation method or it will be there as a note developers. If it is for normal use, I dont think it is a good idea to use tool from the tip of the master.